cli/docker.go

```diff
@@ -397,10 +397,14 @@ func runDockerCVM(ctx context.Context, log slog.Logger, client dockerutil.Docker
 	if err != nil {
 		return xerrors.Errorf("set oom score: %w", err)
 	}
+	ref, err := name.NewTag(flags.innerImage)
+	if err != nil {
+		return xerrors.Errorf("parse ref: %w", err)
+	}
```

Comment:
suggestion: do we have tests covering this newly added validation logic?

- `image`
- `image:latest`
- `image:sometag`
- `repo/image`
- `repo/image:latest`
- `repo/iamge:sometag`

We also use `flags.innerImage` directly on line 488; should we instead use `ref.Name()` there?

Reply:
I think it's ok to just pass the raw `innerImage`. The parsing is just when we need to grab an part of the image (such as the registry).

Reply:
Tests added 👍